bundles/CoreBundle/Resources/config/pimcore/flysystem.yaml

```
@@ -74,4 +78,6 @@ flysystem:
             adapter: 'local'
             visibility: private
             options:
-                directory: '%kernel.project_dir%/var/application-logger'
+                # TODO: revert default to `var/application-logger` when PIMCORE_LOG_FILEOBJECT_DIRECTORY is deprecated
+                directory: "%kernel.project_dir%%log.fileobject%"
```

```suggestion
                directory: "%log.fileobject%"
```
Not 100% sure, but I think we don´t need `%kernel.project_dir%` here. I tried the following:

1.) Checked out 10.4.6 
2.) Added this line `PIMCORE_LOG_FILEOBJECT_DIRECTORY=/var/www/html/var/application-logger-custom/` to my .env file
3.) Started an import with data-importer
4.) Checked the logs tab and the file system, `fileObject` gets stored in `var/application-logger-custom/` as expected

After that:

5.) Checked out this pr
6.) Started the same import again
7.) Checked the filesystem, `fileObject` now gets stored in `var\www\html\var\application-logger-custom\` 

Info: The fallback for `fileObjects` stored before 10.4.6. is still working.

@kingjia90 What do you think? Could you please double check?

---

Just double-checked and you are right, in my local tests i've just used defined the constants as '/var/application-logger-custom', missing that it take a full path.